modules/adding-tab-pods-page.adoc

```diff
@@ -1,13 +1,40 @@
 // Module included in the following assemblies:
 //
-// * web_console/dynamic-plug-ins.adoc
+// * web_console/dynamic-plugin-example.adoc
 
 :_content-type: PROCEDURE
 [id="adding-tab-to-pods-page_{context}"]
 = Adding a tab to the pods page
-The following procedure adds a tab to the *Pod Details* page as an example extension to your plug-in.
 
-.Procedure
+. Visit the link:https://github.com/openshift/console-plugin-template[`console-plugin-template`] repository containing a template for creating plug-ins.
+
+. Select *Use this Template* to create a GitHub repository.
+
+. Re-name the template with the name of your plug-in.
+
+. From your copied repository, clone it to your local machine so you can edit the code.
+
+. Edit the plug-in metadata in the `consolePlugin` declaration of `package.json`.
++
+[source,json]
+----
+"consolePlugin": {
+  "name": "my-plugin", <1>
+  "version": "0.0.1", <2>
+  "displayName": "My Plugin", <3>
+  "description": "Enjoy this shiny, new console plugin!", <4>
+  "exposedModules": {
+    "ExamplePage": "./components/ExamplePage"
+  },
+  "dependencies": {
+    "@console/pluginAPI": "*"
+  }
+}
+----
+<1> Update the name of your plug-in.
+<2> Update the version.
+<3> Update the display name for your plug-in.
+<4> Update the description with a synopsis about your plug-in.
```

Comment:
It's a little odd we don't have them run the plugin first before making any edits. It would be good to show the hello world stuff working before anything else. I think this underscores how easy it is to get running.

Reply:
I added a additional resource link to it before the procedure to edit

Reply:
I think that can work, but it still seems more natural to me to

1. Clone the project
2. Update the plugin metadata
3. Run everything to see it working

This seems particularly true if you use `yarn run start` since it will automatically update the UI when you make the changes below without having to restart your server. The browser will just refresh with your changes.